index.html

```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
             /></a>
           </li>
         </ul>
-      </section>
+    </section>
+    <section class="grid-container">
+      <div class="flex-container">
+        <div class="works-card">
```

Comment:
- [ ] Kindly consider applying the following changes to your design to make it pixel-perfect to Figma design:
- [ ] The body of your project should have `#E5E5E5` color as seen in figma design.
- [ ] The `li`s under the title `CANOPY` are not properly aligned. kindly align them per figma design and also in other projects' cards.
- [ ] The `p` text in the Figma design has a lighter font weight. kindly consider fixing it.
your design
![qwer](https://user-images.githubusercontent.com/96683843/212404205-1f018da8-2521-4e26-9f61-e38f1b866bcf.jpg)


figma design
![qw](https://user-images.githubusercontent.com/96683843/212404069-729e7e3f-9dad-42c3-8301-f4c6f6006b96.jpg)